config/README.md

```diff
@@ -0,0 +1,112 @@
+# Introduction
+
+This document describes the available options in the JSON config for `status-go`.
+
+The structure of the JSON config is defined in the [`params/config.go`](/params/config.go) file, which also contains details comments on meaning of each option.
```

Comment:
I would add that `NodeConfig` is the main struct one should start looking from.

Reply:
Will do.